ui/core/src/model/calculations.ts

```diff
@@ -12,14 +12,14 @@
 // limitations under the License.
 
 import findLast from 'lodash/findLast';
-import meanBy from 'lodash/meanBy';
-import sumBy from 'lodash/sumBy';
+import { default as lodashMean } from 'lodash/mean';
```

Comment:
Renamed this to avoid conflict with the `mean` value below when I switched to the regular `mean` and not `meanBy`.